install/kubernetes/helm/istio/charts/grafana/values.yaml

```diff
@@ -4,7 +4,7 @@
 enabled: false
 replicaCount: 1
 image:
-  repository: grafana/grafana
+  repository: docker.io/grafana/grafana
```

Comment:
The root cause of your troubles with caching the grafana image are here:

https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml#L41

which should be more like:

https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/prometheus/templates/deployment.yaml#L33

IOW, please use `{{ .Values.hub }}` and remove the hardcoded repo in the values.yaml you have added.

Reply:
@sdake thanks for the guidance here. Just to confirm, that would be:

`{{ .Values.hub }}grafana/grafana` <-- without the `/`, is that correct?